concepts/booleans/introduction.md

```diff
@@ -1,3 +1,8 @@
 # Introduction
 
-TODO
+Booleans in Rust are represented by the `bool` type, which values can be either `true` or `false`.
+
+Rust supports six comparison operators: `==` (Equal), `!=` (Not equal), `>`	(Greater than), `<` (Less than), `>=`
+(Greater than or equal to), `<=` (Less than or equal to).
+
+Rust supports two lazy boolean operators: `||` (Or), `&&` (And).
```

Comment:
Maybe add a sentence describing the short-circuiting behavior? Alternately, remove the word "lazy". Otherwise, this could be hard to understand for someone who doesn't already know how these operators operate.